generators/readme/templates/_README.md

```diff
@@ -0,0 +1,194 @@
+
+## License
+<% if (!license.name) { %>
+__*You've elected NOT to include an open source license. If this is an InnerSource product repository that is hosted on private local area network ((e.g., within a company-Intranet), doing without a license may be acceptable.*__
+
+__*However, please consider [the consequences of publishing software products without a license][license-no-license-url] before you release:*__
+
+> Disallowing use of your code might not be what you intend by “no license.” An [open-source license][license-choose-url] allows reuse of your code while retaining copyright. If your goal is to completely opt-out of copyright restrictions, try a [public domain dedication][license-unlicense-url].
+>
+> No License. (n.d.). Retrieved September 13, 2017, from https://choosealicense.com/no-license/
+
+<% } %>
+<% if (license.name) { %>
+[<%= license.name %>][license-url]<% } %> © [<%= author.name %>][author-url].
+
+
+
+
+<!-- ⛔️ 📝 NOTE: PLEASE ALPHABETIZE LINK REFERENCES. 📝 ⛔️ -->
+
+[author-url]: <%= author.url %>
+[license-choose-url]: https://choosealicense.com/
+[license-no-license-url]: https://choosealicense.com/no-license/
+[license-unlicense-url]: https://choosealicense.com/licenses/#unlicense
+[code-of-conduct-url]: ./CODE_OF_CONDUCT.md
+[contributing-url]: ./CONTRIBUTING.md
+[issues-new-defect-url]: <%= repository.url %>/issues/new?title=fix%28affected-scope%29%3A+subject-line-with-very-few-words&labels=Priority%3A+Medium%2CStatus%3A+Review+Needed%2CType%3A+Defect&body=%2A%2A%F0%9F%92%A1+TIP%3A%2A%2A+Select+the+%E2%86%96%EF%B8%8E%E2%8E%BE+Preview+%E2%8F%8B+Tab+above+help+read+these+instructions.%0D%0A%0D%0A%23%23+1.+Issue+type%0D%0A%3E%E2%8C%A6+Type+the+letter+%22x%22+in+the+%22checkbox%22+the+best+describe+this+issue.%0D%0A%0D%0A-+%5Bx%5D+__Feature%3A__+I%27m+requesting+a+product+enhancement.%0D%0A%0D%0A%23%23+2.+User+story+summary%0D%0A%3E%E2%8C%A6+Describe+what+you+want+to+accomplish%2C+in+what+role%2Fcapacity%2C+and+why+it%27s+important+to+you.%0D%0A%0D%0A%3E+__EXAMPLE%3A__%0D%0A%3E+As+a+Applicant%2C%0D%0A%3E+I+want+to+submit+my+resume%0D%0A%3E+In+order+to+be+considered+for+a+job+opening.%0D%0A%0D%0AAs+a+%7Brole%7D%2C%0D%0AI+must%2Fneed%2Fwant%2Fshould+%7Bdo+something%7D%0D%0AIn+order+to+%7Bachieve+value%7D.%0D%0A%0D%0A%23%23+3.+Acceptance+criteria%0D%0A%3E%E2%8C%A6+Replace+the+examples+below+with+your+own+imperative%2C+%22true%2Ffalse%22+statements+for+the+__behavior+you+expect__+to+see%2C+or+the+behavior+that+__would__+be+true+if+there+were+no+errors+%28for+defects%29.%0D%0A%0D%0A-+%5B+%5D+1.+Job+Applicants+receive+a+confirmation+email+after+they+submit+their+resumes.%0D%0A-+%5B+%5D+2.+An+Applicant%27s+resume+information+isn%27t+lost+when+errors+occur.%0D%0A-+%5B+%5D+3.+%7Bcriterion-three%7D%0D%0A-+%5B+%5D+4.+%7Bcriterion-four%7D%0D%0A%0D%0A%3C%21--+%E2%9B%94%EF%B8%8F++Do+not+remove+anything+below+this+comment.+%E2%9B%94%EF%B8%8F++--%3E%0D%0A%5Bicon-info-image%5D%3A+..%2Fdocs%2Fimg%2Ficons8%2Ficon-info-50.png%0D%0A
```

Comment:
This beast is throwing an error when being toggled in the CLI

Reply:
<details>
    <summary>Super long error</summary>

```
TypeError: /Users/sh/_code/github/generator-community/generators/readme/templates/_README.md:188
    186| [code-of-conduct-url]: ./CODE_OF_CONDUCT.md
    187| [contributing-url]: ./CONTRIBUTING.md
 >> 188| [issues-new-defect-url]: <%= repository.url %>/issues/new?title=fix%28affected-scope%29%3A+subject-line-with-very-few-words&labels=Priority%3A+Medium%2CStatus%3A+Review+Needed%2CType%3A+Defect&body=%2A%2A%F0%9F%92%A1+TIP%3A%2A%2A+Select+the+%E2%86%96%EF%B8%8E%E2%8E%BE+Preview+%E2%8F%8B+Tab+above+help+read+these+instructions.%0D%0A%0D%0A%23%23+1.+Issue+type%0D%0A%3E%E2%8C%A6+Type+the+letter+%22x%22+in+the+%22checkbox%22+the+best+describe+this+issue.%0D%0A%0D%0A-+%5Bx%5D+__Feature%3A__+I%27m+requesting+a+product+enhancement.%0D%0A%0D%0A%23%23+2.+User+story+summary%0D%0A%3E%E2%8C%A6+Describe+what+you+want+to+accomplish%2C+in+what+role%2Fcapacity%2C+and+why+it%27s+important+to+you.%0D%0A%0D%0A%3E+__EXAMPLE%3A__%0D%0A%3E+As+a+Applicant%2C%0D%0A%3E+I+want+to+submit+my+resume%0D%0A%3E+In+order+to+be+considered+for+a+job+opening.%0D%0A%0D%0AAs+a+%7Brole%7D%2C%0D%0AI+must%2Fneed%2Fwant%2Fshould+%7Bdo+something%7D%0D%0AIn+order+to+%7Bachieve+value%7D.%0D%0A%0D%0A%23%23+3.+Acceptance+criteria%0D%0A%3E%E2%8C%A6+Replace+the+examples+below+with+your+own+imperative%2C+%22true%2Ffalse%22+statements+for+the+__behavior+you+expect__+to+see%2C+or+the+behavior+that+__would__+be+true+if+there+were+no+errors+%28for+defects%29.%0D%0A%0D%0A-+%5B+%5D+1.+Job+Applicants+receive+a+confirmation+email+after+they+submit+their+resumes.%0D%0A-+%5B+%5D+2.+An+Applicant%27s+resume+information+isn%27t+lost+when+errors+occur.%0D%0A-+%5B+%5D+3.+%7Bcriterion-three%7D%0D%0A-+%5B+%5D+4.+%7Bcriterion-four%7D%0D%0A%0D%0A%3C%21--+%E2%9B%94%EF%B8%8F++Do+not+remove+anything+below+this+comment.+%E2%9B%94%EF%B8%8F++--%3E%0D%0A%5Bicon-info-image%5D%3A+..%2Fdocs%2Fimg%2Ficons8%2Ficon-info-50.png%0D%0A
    189| [issues-new-feat-url]: <%= repository.url %>/issues/new?title=feat%28affected-scope%29%3A+subject-line-with-very-few-words&labels=Priority%3A+Medium%2CStatus%3A+Review+Needed%2CType%3A+Feature&body=%2A%2A%F0%9F%92%A1+TIP%3A%2A%2A+Select+the+%E2%86%96%EF%B8%8E%E2%8E%BE+Preview+%E2%8F%8B+Tab+above+help+read+these+instructions.%0D%0A%0D%0A%23%23+1.+Issue+type%0D%0A%3E%E2%8C%A6+Type+the+letter+%22x%22+in+the+%22checkbox%22+the+best+describe+this+issue.%0D%0A%0D%0A-+%5Bx%5D+__Feature%3A__+I%27m+requesting+a+product+enhancement.%0D%0A%0D%0A%23%23+2.+User+story+summary%0D%0A%3E%E2%8C%A6+Describe+what+you+want+to+accomplish%2C+in+what+role%2Fcapacity%2C+and+why+it%27s+important+to+you.%0D%0A%0D%0A%3E+__EXAMPLE%3A__%0D%0A%3E+As+a+Applicant%2C%0D%0A%3E+I+want+to+submit+my+resume%0D%0A%3E+In+order+to+be+considered+for+a+job+opening.%0D%0A%0D%0AAs+a+%7Brole%7D%2C%0D%0AI+must%2Fneed%2Fwant%2Fshould+%7Bdo+something%7D%0D%0AIn+order+to+%7Bachieve+value%7D.%0D%0A%0D%0A%23%23+3.+Acceptance+criteria%0D%0A%3E%E2%8C%A6+Replace+the+examples+below+with+your+own+imperative%2C+%22true%2Ffalse%22+statements+for+the+__behavior+you+expect__+to+see%2C+or+the+behavior+that+__would__+be+true+if+there+were+no+errors+%28for+defects%29.%0D%0A%0D%0A-+%5B+%5D+1.+Job+Applicants+receive+a+confirmation+email+after+they+submit+their+resumes.%0D%0A-+%5B+%5D+2.+An+Applicant%27s+resume+information+isn%27t+lost+when+errors+occur.%0D%0A-+%5B+%5D+3.+%7Bcriterion-three%7D%0D%0A-+%5B+%5D+4.+%7Bcriterion-four%7D%0D%0A%0D%0A%3C%21--+%E2%9B%94%EF%B8%8F++Do+not+remove+anything+below+this+comment.+%E2%9B%94%EF%B8%8F++--%3E%0D%0A%5Bicon-info-image%5D%3A+..%2Fdocs%2Fimg%2Ficons8%2Ficon-info-50.png%0D%0A
    190| [license-url]: ./LICENSE
    191| [nodejs-url]: https://nodejs.org

Cannot read property 'url' of undefined
    at eval (eval at compile (/Users/sh/_code/github/generator-community/node_modules/ejs/lib/ejs.js:549:12), <anonymous>:95:36)
    at returnedFn (/Users/sh/_code/github/generator-community/node_modules/ejs/lib/ejs.js:580:17)
    at Object.exports.render (/Users/sh/_code/github/generator-community/node_modules/ejs/lib/ejs.js:384:37)
    at process (/Users/sh/_code/github/generator-community/node_modules/mem-fs-editor/lib/actions/copy-tpl.js:12:18)
    at applyProcessingFunc (/Users/sh/_code/github/generator-community/node_modules/mem-fs-editor/lib/actions/copy.js:13:16)
    at EditionInterface.exports._copySingle (/Users/sh/_code/github/generator-community/node_modules/mem-fs-editor/lib/actions/copy.js:63:16)
    at files.forEach.file (/Users/sh/_code/github/generator-community/node_modules/mem-fs-editor/lib/actions/copy.js:50:10)
    at Array.forEach (native)
    at EditionInterface.exports.copy (/Users/sh/_code/github/generator-community/node_modules/mem-fs-editor/lib/actions/copy.js:49:9)
    at EditionInterface.module.exports [as copyTpl] (/Users/sh/_code/github/generator-community/node_modules/mem-fs-editor/lib/actions/copy-tpl.js:10:8)
```
</details>

Reply:
Man, this is a pain in the :end:. I created a new issue (#4) to address handling directories (repository root directories) that don't have a `package.json` file already. I also included instructions in the README.md.

I guess it'd be best to either

1. Create a `package.json` if one doesn't yet exist, or
1. Abort generation and request they run `npm init`, or
1. Offer to run `npm init` on their behalf.

Thoughts, @spentacular ?

Reply:
I'd guess 1? I think the cycle would be weirder if you did that first since you would potentially need to re-init if you haven't specified a remote, since those are blank

Reply:
You're right. I'll label #2 and #3 with "Status: Blocked" and start working on #4.